src/Gui/Hooks/useStoryPreview/StoryPreviewer.luau

```diff
@@ -0,0 +1,186 @@
+--!nonstrict
```

Comment:
As an addendum to my previous comment, it's okay to keep these classes as nonstrict because Luau OOP isn't good